mirror-godot-app/ui/common/shaders/dark_blue_blur.mat.tres

```diff
@@ -4,6 +4,3 @@
 
 [resource]
 shader = ExtResource("1_5syw1")
-shader_parameter/blur = 2.417
```

Comment:
We need to completely depreciate use of the blur shader, it causes problems on intel and on the quest 2/3